03-a_review_of_r_modeling_fundamentals.Rmd

```diff
@@ -58,7 +58,7 @@ trees %>%
   corrr::correlate()
 ```
 
-Shame on you `r emo::ji('wink')` if you didn't guess I would make a scatter plot given a data set with two variables.
+Shame on you 😉 if you didn't guess I would make a scatter plot given a data set with two variables.
```

Comment:
Can you change the smiley back to `r emo::ji('wink')`? I believe it will keep it more friendly across OSes.